src/subsys/ngx_subsys_lua_module.c.tt2

```diff
@@ -1023,6 +1030,7 @@ ngx_[% subsys %]_lua_create_main_conf(ngx_conf_t *cf)
      */
 
     lmcf->pool = cf->pool;
+    lmcf->load_resty_core = NGX_CONF_UNSET;
```

Comment:
We should give `load_resty_core` a proper default value  (`1` in this case)in `ngx_[% subsys %]_lua_init_main_conf`. Despite this current version works, using `NGX_CONF_UNSET` in the `if` below is really a hack and breaks the semantics of the flag and should be avoided.